spec/features/goal_tool_spec.rb

```diff
@@ -15,7 +15,7 @@
   end
 
   scenario "Displays due date" do
-    expect(page).to have_content "Due: #{Date.today.to_s(:date)}"
+    expect(page).to have_content "Due: #{Date.today.strftime("%b %d %Y")}"
```

Comment:
This is a good question, if I use the method, then don't I fall into the problem that I testing the method by using the method?